app/views/transportMeans/active/IdentificationNumberView.scala.html

```diff
@@ -18,21 +18,22 @@
 @import controllers.transportMeans.active.routes
 @import models.LocalReferenceNumber
 @import viewModels.components.InputTextViewModel._
-@import views.html.components.{Button, ErrorSummary, InputText}
+@import views.html.components.{Button, ErrorSummary, InputText, InsetText}
 
 @this(
     mainTemplate: MainTemplate,
     formHelper: FormWithCSRF,
     inputText: InputText,
+    insetText: InsetText,
     errorSummary: ErrorSummary,
     button: Button,
     phaseConfig: PhaseConfig
 )
 
-@(form: Form[String], lrn: LocalReferenceNumber, mode: Mode, activeIndex: Index, prefix: String, args: String*)(implicit request: Request[_], messages: Messages)
+@(form: Form[String], lrn: LocalReferenceNumber, mode: Mode, activeIndex: Index, prefix: String, identificationNumber: String)(implicit request: Request[_], messages: Messages)
```

Comment:
Looks like we don't need to pass prefix in anymore and can just define it statically in the controller and view as is our standard pattern